### PR TITLE
refactor: Update MWAA blueprint for eks-blueprints v5

### PR DESCRIPTION
**Pull request by askulkarni2**
### What does this PR do?

This PR updates the MWAA blueprint for the upcoming v5 changes in EKS Blueprints.

### More

- [x] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ ] Mandatory for new blueprints. Yes, I have added a example to support my blueprint PR
- [ ] Mandatory for new blueprints. Yes, I have updated the `website/docs` or `website/blog` section for this feature
- [x] Yes, I ran `pre-commit run -a` with this PR. Link for installing [pre-commit](https://pre-commit.com/) locally

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: schedulers/terraform/managed-airflow-mwaa/main.tf [hunk 1]
```diff
@@ -3,8 +3,8 @@
 #------------------------------------------------------------------------
 
 module "mwaa" {
-  source  = "aws-ia/mwaa/aws"
-  version = "0.0.1"
+  source = "github.com/askulkarni2/terraform-aws-mwaa?ref=fix%2Fprovider-versions"
```
Comment (vara-bonthu) [inline schedulers/terraform/managed-airflow-mwaa/main.tf:6]: Shall we hold on to the PR until the versions fix is applied to `aws-ia/mwaa/aws` module and update this source and version?

File: website/docs/job-schedulers/aws-managed-airflow.md [hunk 2]
```diff
@@ -52,7 +57,10 @@ The following components are provisioned in your environment:
   - An EMR virtual cluster registered with the newly created EKS
   - A S3 bucket with DAG code
```
Comment (vara-bonthu) [inline website/docs/job-schedulers/aws-managed-airflow.md:59]: Here we can also mention about our install.sh script which can be used alternatively or update this section

**Review by vara-bonthu — COMMENTED** [review]
Thanks @askulkarni2 👍🏼 One minor comment

File: schedulers/terraform/managed-airflow-mwaa/main.tf (hunk 1 above)
Comment (askulkarni2) [reply schedulers/terraform/managed-airflow-mwaa/main.tf:6]: Yeah we should hold. That updated MWAA PR is up.

Comment (vara-bonthu) [reply schedulers/terraform/managed-airflow-mwaa/main.tf:6]: I have published a new tag for the recent merge on MWAA. You can use `0.0.2` and update the PR.

**Comment by vara-bonthu** [comment]
Closing this PR as this is upgraded by @bbgu1 in his recent PR #248

**Pull request closed without merging**